setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="ocp-models",
+    version="0.0.1",
+    description="A machine learning package for molecular systems involved with catalysis",
+    url="https://github.com/Open-Catalyst-Project/baselines",
+    author="Abhishek Das, Kevin Tran, Junwoong Yoon, Brandon Wood, and others",
```

Comment:
I have a preference towards not putting down a list of authors here for now if that's okay?